operations/mimir-mixin/alerts/blocks.libsonnet

```diff
@@ -223,10 +223,10 @@
           // Alert if the bucket index has not been updated for a given user. The default update interval is 900 seconds
           // so we alert if we've missed two updates plus a 300 second buffer to avoid false-positives. It's important
           // that this alert fire before queriers start to return errors because the bucket index is too old (3600 seconds
-          // by default).
+          // by default). Allow a 15m lookback to include compactors that may have recently updated the index and then been terminated.
```

Comment:
did you choose 15m because of the default cleanup interval? Could be worth mentioning this in the comment